### DISCUSSION TITLE
Adding a distro to FastFetch

### DISCUSSION THREAD

**Discussion by YINGVAY** [fastfetch-cli/fastfetch#1296] [Ideas]
Me and my team are currently working on a new distro built off of NixOS called "MidOS", I would love to speak with you about possibly adding our distro to the the list of logos you currently have, feel free to contact me and my team at this email.

d.hendo308@gmail.com

Also, here is a link to the distro its self, its a massively fast growing distro and we hope to go mainstream as soon as possible!!
https://codeberg.org/midirhee12/mid-layer

**Comment by CarterLi** [comment]
https://github.com/fastfetch-cli/fastfetch/issues/new?assignees=&labels=logo+request&projects=&template=logo_request.md&title=%5BLOGO%5D+
